ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -488,3 +488,11 @@ def __iter__(self):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self, value, out=self)
+    
+    def ptp(self, *, axis=None, out=None, keepdims=False):
+        if isinstance(self._ivy_array, int):
```

Comment:
I don't think it makes sense to flatten the array if it's an integer. Maybe this check is not needed at all.

Reply:
GREAT, 
Finally, all test passed! 👯 
please review my pull request :)